proxy-robots-module/src/main/java/orwell/proxy/config/Configuration.java

```diff
@@ -10,11 +10,11 @@
 import java.io.InputStream;
 
 public class Configuration implements IConfiguration {
-    final static Logger logback = LoggerFactory.getLogger(Configuration.class);
+    private final static Logger logback = LoggerFactory.getLogger(Configuration.class);
     private final String filePath;
     private final EnumConfigFileType enumConfigFileType;
     public boolean isPopulated = false;
```

Comment:
It is used outside the class to check the configuration has been properly populated. 
This should rather be a public getter with no setter, probably.